components/hab/src/command/plan/init.rs

```diff
@@ -115,10 +117,65 @@ pub fn start(ui: &mut UI,
                your habitat. There are several hooks to create and tweak! See the full list \
                with info here: https://www.habitat.sh/docs/reference/plan-syntax/#hooks"));
 
+    try!(render_ignorefile(ui, &root));
+
     try!(ui.end("A happy abode for your code has been initialized! Now it's time to explore!"));
     Ok(())
 }
 
+fn render_ignorefile(ui: &mut UI, root: &str) -> Result<()> {
+    let parent = format!("{}/..", root);
+    let expanded = canonicalize(&parent)?;
+    let current_path = Path::new(&expanded);
+
+    fn is_git_managed(path: &Path) -> bool {
```

Comment:
While it's _valid_ to define a function within another, I would recommend doing this outside of the `render_ignorefile()` function